conf/Nanopore-Jun2020-Automation.conf

```diff
@@ -0,0 +1,88 @@
+# DO NOT USE THIS FILE IF YOU HAVE READS CREATED BY A
+# GUPPY VERSION OLDER THAN 3.6.0.
+
+# This file contains Shasta options that, as of June 2020,
```

Comment:
Let's rename this file `Nanopore-Sep2020.conf`, without the "Automation" suffix. Instead, put a well visible comment near the top of the file to describe what it does. I plan to add a section on `ReadGraph.creationMethod 2` to the documentation.